pkg/osvscanner/osvscanner.go

```diff
@@ -888,13 +888,14 @@ func patchPackageForRequest(pkg scannedPackage) scannedPackage {
 
 func makeRequest(
 	r reporter.Reporter,
-	packages []scannedPackage,
+	packages *[]scannedPackage,
```

Comment:
Updated it to a pointer as we modified this slice (stdlib package version https://github.com/google/osv-scanner/pull/740) in this function.

Reply:
I don't think this needs to be a pointer unless we are adding/removing items from the slice. We can access and directly set the items in the loop, as a slice is already a pointer to the elements within the slice.

Reply:
Oh yeah. I misunderstood the purpose of this changes. Reverted changes back. Thanks!